components/scream/src/physics/rrtmgp/tests/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ endif()
 
 # Required libraries
 find_library(NETCDF_C netcdf HINTS ${NetCDF_C_PATHS}/lib)
-set (NEED_LIBS ${NETCDF_C} rrtmgp scream_rrtmgp scream_share yakl)
+set (NEED_LIBS ${NETCDF_C} rrtmgp scream_rrtmgp scream_share yakl share_util)
```

Comment:
P.S: I don't know how to comment a line not involved in the PR, but I kind of want to: below I see

```
set_target_properties(generate_baseline PROPERTIES COMPILE_FLAGS "${YAKL_CXX_FLAGS}")
```
This will tell CMake to add ${YAKL_CXX_FLAGS} to the compile flags of _all languages_. I assume some of those flags might be c++ specific. In this case, one should use generator expressions (and also, use target_compile_options, which is the "cmake-y" way for this stuff):
```
target_compile_options(generate_baselines PUBLIC $<$<COMPILE_LANGUAGE:CXX>:${YAKL_CXX_FLAGS}>)
```

Reply:
Thanks Luca. This should all be fixed (including the suggestion about using the generator expressions).